src/main/java/io/electrum/airtime/api/PurchaseResource.java

```diff
@@ -156,6 +164,33 @@ public final void reversePurchase(
             .reversePurchase(body, securityContext, request, httpHeaders, asyncResponse, uriInfo, httpServletRequest);
    }
 
+   /**
+    * @since 5.14.0
+    */
+   @POST
+   @Path(LookupPurchase.RELATIVE_PATH)
+   @Consumes({ "application/json" })
+   @Produces({ "application/json" })
+   @ApiOperation(nickname = LookupPurchase.LOOKUP_PURCHASE, value = "Look up the status of a previously submitted purchase request.", notes = "If no response was received to a purchase request due to a timeout or temporary communications "
+         + "failure, PoS may lookup the same purchase request by calling this resource.")
```

Comment:
It might not be a POS so how about "client instead"?
```suggestion
         + "failure, the client may lookup the same purchase request by calling this resource.")
```